### DISCUSSION TITLE
Is it possible to hide rows before save into pdf?

### DISCUSSION THREAD

**Discussion by mrmarcin12** [PHPOffice/PhpSpreadsheet#2999] [Q&A] [answered]
When I try to use function RemoveRow I getting fatal error. When I set "setWidth(0)" its not working and no effect. Is it possible to delete or hide row before save into pdf?

It looks like this and dont working:

>             $offer->getActiveSheet()->getRowDimension(43)->setRowHeight(10);
>             $offer->getActiveSheet()->getRowDimension(44)->setRowHeight(1, 'px');
>             $offer->getActiveSheet()->getRowDimension('43')->setCollapsed(true);
>             $offer->getActiveSheet()->getRowDimension('44')->setVisible(false);
> 
> 
>         $pdfWriter = \PhpOffice\PhpSpreadsheet\IOFactory::createWriter($offer,'Mpdf');
>         $pdfWriter->save('filename.pdf');

**Comment by oleibman** [comment]
```php
$sheet->getRowDimension($rowNumber)->setVisible(false);
```
This is essentially what you are doing. It works correctly for xlsx, html, and dompdf. It does not seem to work for mpdf. I'll see if I can raise an issue there.

**Reply by oleibman** [reply]
I have opened Mpdf issue https://github.com/mpdf/mpdf/issues/1696

**Reply by mrmarcin12** [reply, marked as answer]
Thank you, with dompdf working good.
